specifyweb/frontend/js_src/lib/components/Formatters/Fields.tsx

```diff
@@ -25,9 +25,11 @@ import type { Formatter } from './spec';
 export function Fields({
   table,
   fields: [fields, setFields],
+  isReadOnly,
```

Comment:
don't pass read-only as a prop. use this instead:

```ts
const isReadOnly = React.useContext(ReadOnlyContext);
```

by using ReadOnlyContext everywhere, every component is able to handle being used in a read-only environment in a consistent way - better for code reuse too